src/services/identity/DeploymentClient.ts

```diff
@@ -61,6 +64,11 @@ class DeploymentClient {
       this.amplifyClient.send(new CreateBranchCommand(options))
     ) as ResultAsync<CreateBranchCommandOutput, AmplifyError>
 
+  sendUpdateApp = (options: UpdateBranchCommandInput) =>
+    wrap(
+      this.amplifyClient.send(new UpdateBranchCommand(options))
+    ) as ResultAsync<UpdateBranchCommandOutput, AmplifyError>
```

Comment:
```suggestion
    wrap(this.amplifyClient.send(new CreateAppCommand(options)))

  sendCreateBranch = (options: CreateBranchCommandInput) =>
    wrap(this.amplifyClient.send(new CreateBranchCommand(options)))

  sendUpdateApp = (options: UpdateBranchCommandInput) =>
    wrap(this.amplifyClient.send(new UpdateBranchCommand(options)))
```

Reply:
just need to change
```ts
const wrap = <T>(promise: Promise<T>)
```

Reply:
Actually I've swapped this back - wrapping alone doesn't seem to provide us with the correct return type, which is breaking some existing functionality